flexmeasures/ui/templates/crud/accounts.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %} {% set active_page = "accounts" %} {% block title %} Account listing {% endblock %} {% block
+divs %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-2"></div>
+        <div class="col-sm-8">
+            <h3>All Accounts
+            </h3>
+            <div class="card"></div>
+            <table class="table table-striped table-responsive paginate">
+                <thead>
+                    <tr>
+                        <th>Account</th>
+                        <th>ID</th>
+                        <th>Roles</th>
+
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for account in accounts %}
+                    <tr>
+                        <td>
+                            <a href="/accounts/{{ account.id }}" title="View this account">{{ account.name }}</a>
+                        </td>
+                        <td>
+                            {{ account.id }}
+                        </td>
+                        <td>
+                            {{ account.account_roles | join(", ") }}
```

Comment:
I believe this concatenates the role IDs, while we need the role names. Maybe use `account.account_roles.values()`?

Reply:
The API doesn't return the account_role names currently, only the ID. I think that should be a separate issue if we want to add it.

Reply:
[This PR](https://github.com/FlexMeasures/flexmeasures/pull/625) should unblock you. If you approve it, I'll (squash) merge it into this PR branch so you can continue.

Reply:
Thanks for PR625 @Flix6x , added the roles to the accounts tables